dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -10,6 +10,24 @@
 #include "Utils.h"
 
 static constexpr wstring_view c_navigationViewItemPresenterName = L"NavigationViewItemPresenter"sv;
+static constexpr auto c_repeater = L"NavigationViewItemMenuItemsHost"sv;
+static constexpr auto c_rootGrid = L"NVIRootGrid"sv;
+static constexpr auto c_flyoutContentGrid = L"FlyoutContentGrid"sv;
+static constexpr auto c_flyoutTitleContentPresenterGrid = L"FlyoutTitleContentPresenterGrid"sv;
+static constexpr auto c_flyoutTitleContentPresenter = L"FlyoutTitleContentPresenter"sv;
+
+// Visual States
+static constexpr auto c_pressedSelected = L"PressedSelected"sv;
+static constexpr auto c_pointerOverSelected = L"PointerOverSelected"sv;
+static constexpr auto c_selected = L"Selected"sv;
+static constexpr auto c_pressed = L"Pressed"sv;
+static constexpr auto c_pointerOver = L"PointerOver"sv;
+static constexpr auto c_disabled = L"Disabled"sv;
+static constexpr auto c_enabled = L"Enabled"sv;
+static constexpr auto c_normal = L"Normal"sv;
+static constexpr auto c_chevronHidden = L"ChevronHidden"sv;
+static constexpr auto c_chevronVisible = L"ChevronVisible"sv;
```

Comment:
I think ideally you would include visual state names with these definitions

Reply:
Sorry, I don't get what you mean? These are visual state names.

Reply:
For instance you don't have these state names up here L"IconOnLeft"  L"IconOnly" L"ContentOnly" names up bere

---
In reply to: [394637672](https://github.com/microsoft/microsoft-ui-xaml/pull/2004#discussion_r394637672) [](ancestors = 394637672)